### ISSUE TITLE
Question: Unable to install archivebox in a Digital Ocean Ubuntu machine

### ISSUE THREAD

**Issue by raknahs1991** [ArchiveBox/ArchiveBox#629]
I've installed all the dependencies, but when I run

`curl https://raw.githubusercontent.com/pirate/ArchiveBox/master/bin/setup.sh | sh`

I get this message saying archivebox not found. What do I do?

```
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0[i] ArchiveBox Setup Script 📦

    This is a helper script which installs the ArchiveBox dependencies on your system using homebrew/aptitude.
    You may be prompted for a password in order to install the following:

        - git
        - python3, python3-pip, python3-distutils
        - curl
        - wget
        - youtube-dl
        - chromium-browser  (skip this if Chrome/Chromium is already installed)

    If you'd rather install these manually, you can find documentation here:
        https://github.com/ArchiveBox/ArchiveBox/wiki/Install

Press enter to continue with the automatic install, or Ctrl+C to cancel...
sh: 22: read: arg count

100  4244  100  4244    0     0  25721      0 --:--:-- --:--:-- --:--:-- 25721
[+] Updating apt repos...
Reading package lists...
E: Could not open lock file /var/lib/apt/lists/lock - open (13: Permission denied)
E: Unable to lock directory /var/lib/apt/lists/
W: Problem unlinking the file /var/cache/apt/pkgcache.bin - RemoveCaches (13: Permission denied)
W: Problem unlinking the file /var/cache/apt/srcpkgcache.bin - RemoveCaches (13: Permission denied)
[+] Installing python3, wget, curl...
E: Could not open lock file /var/lib/dpkg/lock-frontend - open (13: Permission denied)
E: Unable to acquire the dpkg frontend lock (/var/lib/dpkg/lock-frontend), are you root?
/usr/bin/chromium-browser
[i] chromium-browser already installed, using existing installation.

Chromium 87.0.4280.66 Built on Ubuntu , running on Ubuntu 18.04
Collecting archivebox
  Could not find a version that satisfies the requirement archivebox (from versions: )
No matching distribution found for archivebox

[*] Checking installed versions:
---------------------------------------------------
/usr/bin/python3
Python 3.6.9

/usr/bin/git
git version 2.17.1

/usr/bin/wget
GNU Wget 1.19.4 built on linux-gnu.

/usr/bin/curl
curl 7.58.0 (x86_64-pc-linux-gnu) libcurl/7.58.0 OpenSSL/1.1.1 zlib/1.2.11 libidn2/2.0.4 libpsl/0.19.1 (+libidn2/2.0.4) nghttp2/1.30.0 librtmp/2.3

/usr/bin/youtube-dl
2018.03.14
---------------------------------------------------
sh: 107: archivebox: not found
---------------------------------------------------
[X] Failed to install some dependencies! ‼️
    - Try the Manual Setup instructions in the README.md
    - Try the Troubleshooting: Dependencies instructions in the README.md
    - Open an issue on github to get help: https://github.com/ArchiveBox/ArchiveBox/issues
```

**Comment by evanfarrar** [comment]
Hello! I am just a fellow user but I thought I'd try to support you. It seems that the default Ubuntu that Digital Ocean points you to is 18.04. This prevents archivebox from installing via pip, because archivebox requires [Python >=3.7](https://pypi.org/project/archivebox/). However, you could install the package if you instead booted a droplet with Ubuntu 20.04, a newer version that comes with a newer Python.

Well, you could theoretically install it if you had all of the apt based dependencies installed, but in fact you _couldn't_ if you are relying on the install script to install the `apt` dependencies. There is a small bug in that part of the install script. I may look at opening a pull request to resolve this.

**Comment by pirate** [comment]
Whoops ignore the close and reopen, try the install script on `dev`:

`curl https://raw.githubusercontent.com/ArchiveBox/ArchiveBox/dev/bin/setup.sh | sh`

**Comment by evanfarrar** [comment]
yeah the `apt install -y` is definitely what it took for me to get it working, and also had to use a different chromium package

**Comment by evanfarrar** [comment]
Can confirm, it installs on DO with the script on `dev`

**Comment by pirate** [comment]
`dev` is released as v0.5.4, should be fixed as soon as the apt build goes live.
